src/libs/installer/eventlogger.cpp

```diff
@@ -58,7 +58,16 @@ void EventLogger::initialize(eve_launcher::application::Application_Region regio
     s_provider = provider;
     s_providerName = providerName;
     s_initSuccessful = true;
-    s_gatewayUrl = getGatewayUrl();
+
+    if (QInstaller::isLocalEndpointEnabled())
+    {
+        s_gatewayUrl = QString::fromLatin1("https://localhost:5001/weatherforecast");
```

Comment:
Should this value be configurable via the command line argument? `--local-endpoint=https://...`

Reply:
I thought about it, but thought nah, nobody is going to use this, but then again, could be useful, I'll add that.
Only downside of merging this entire PR, is it allows users to bypass sending the data to the gateway. I could always send it to both places 🤔 (just thought that would be overkill)

Reply:
Also this would always require us to provide the endpoint (that's just how the Qt implementation works).
Or we could introduce an additional parameter for providing the endpoint url `--local-endpoint-url=https://...`